pkg/core/intl/match.go

```diff
@@ -0,0 +1,56 @@
+package intl
+
+import (
+	"golang.org/x/text/language"
+)
+
+func SortSupported(supportedLanguageTags []string, fallbackLanguageTag string) []string {
+	if fallbackLanguageTag == "" {
+		fallbackLanguageTag = DefaultLanguage
+	}
+
+	fallbackIdx := -1
+	for i, tag := range supportedLanguageTags {
+		if tag == fallbackLanguageTag {
+			fallbackIdx = i
+			break
+		}
+	}
+
+	if fallbackIdx < 0 {
+		var s []string
+		s = append(s, fallbackLanguageTag)
```

Comment:
I think `SortSupported` should only sort the list but not adding `fallbackLanguageTag`? If `fallbackLanguageTag` is not one of the supported language, and if the matched result return `fallbackLanguageTag`, render might have problem.

Reply:
The property of `SortSupported` must uphold is that the returned slice is always non-empty and the first element must be `fallbackLanguageTag `. Maybe I should rename the function.

Reply:
But what if user provide `fallbackLanguageTag` not in the `supportedLanguageTags`?

Maybe what we should do is to ensure is default language is in `supportedLanguageTags` and `fallbackLanguageTag` default is default language?

Reply:
> But what if user provide fallbackLanguageTag not in the supportedLanguageTags?

It is clearly a configuration problem introduced by the developer. Skygear only comes with english. If the developer change the `fallback_language` to something else, it is expected that they also have to provide ALL translation of that fallback language.

The purpose of `SortSupported` is to create a string slice that put the fallback language as the first element so that the matcher can use that language as fallback.

Reply:
> It is clearly a configuration problem introduced by the developer. Skygear only comes with english. If the developer change the fallback_language to something else, it is expected that they also have to provide ALL translation of that fallback language.

ok, then keep it as it.